### ISSUE TITLE
[1.16.4] CTD: ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by Sandriell** [ldtteam/minecolonies#6509]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.4
- MineColonies Version: minecolonies-0.13.609-ALPHA-universal
- Structurize Version: structurize-0.13.113-ALPHA-universal
- Related Mods and their Versions: 

## Expected behavior
No crash

## Actual behavior
Game crashed to desktop shortly after loading world.

## Steps to reproduce the issue

Basically crashed shortly after loading, I was just standing around not doing anything yet.

### Logs

- crashlog: https://gist.github.com/Sandriell/eb037ae10d7c1dd46ad2128d630b537d

### Notes

Tested that I could reload and keep playing. Went 10 minutes without any more crashes. Will update of course if it happens again.

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Sandriell** [comment]
Crashed again, second crash report incase there is any different info: https://gist.github.com/Sandriell/841c511813f11d006270121173a29844

Going to revert to an older version for now.

**Comment by Raycoms** [comment]
hotfix pushed
